core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb

```diff
@@ -0,0 +1,13 @@
+if Bundler.locked_gems.dependencies['solidus_stripe']
+
+  generate 'solidus_paypal_commerce_platform:install'
+else
+
+  if @selected_frontend == 'starter'
+    bundle_command "add solidus_stripe --github='solidusio/solidus_stripe'"
+    generate "solidus_paypal_commerce_platform:install --migrate=#{!!options[:migrate]}"
```

Comment:
```suggestion
    generate "solidus_stripe:install --migrate=#{!!options[:migrate]}"
```

Reply:
Why the `migrate` option?

Reply:
Nevermind it was related to the wrong installer being triggered